memestra/nbmemestra.py

```diff
@@ -3,8 +3,7 @@
 from collections import namedtuple
 import nbformat
 
-
-def nbmemestra(nbfile, decorator):
+def nbmemestra(nbfile, decorator, _):
```

Comment:
as nbfile can also be a stringio, there's no reason to ignore the filepath parameter, right?

Reply:
You mean, that `nbmemestra` can be called programmatically and receive a StringIO? And I should check instead of using `nbfile.name` straight away and see if I should use the 3rd parameter or not.